docs/api/RESTAPI.md

```diff
@@ -864,7 +864,7 @@ Result for an individual workflow returned by a workflow query
 |**end**  <br>*optional*|Workflow end datetime|string (date-time)|
 |**id**  <br>*required*|Workflow ID|string|
 |**name**  <br>*required*|Workflow name|string|
-|**start**  <br>*required*|Workflow start datetime|string (date-time)|
+|**start**  <br>*optional*|Workflow start datetime|string (date-time)|
```

Comment:
@calbach Thanks for your contribution! 
For future notice, we generate the `RESTAPI.md` file from the `cromwell.yaml` file by running `sbt generateRestApiDocs` in the Cromwell top-level directory.

If you had only updated the `RESTAPI.md` without updating `cromwell.yaml`, your changes would be overwritten. But since you updated both files your edits will not be overwritten.